board/safety/safety_volkswagen_mqb.h

```diff
@@ -166,6 +214,39 @@ static int volkswagen_mqb_tx_hook(CANPacket_t *to_send, bool longitudinal_allowe
     }
   }
 
+  // Safety check for both ACC_06 and ACC_07 acceleration requests
+  // To avoid floating point math, scale upward and compare to pre-scaled safety m/s2 boundaries
+  if ((addr == MSG_ACC_06) || (addr == MSG_ACC_07)) {
+    bool violation = 0;
```

Comment:
use logical values for bools. I know a lot of panda doesn't do this, but i'd like to move towards this
```suggestion
    bool violation = false;
```